packages/dfw/src/dfwcomponents/WorkOrderCell/WorkOrderCell.tsx

```diff
@@ -50,23 +53,28 @@ export const WorkOrderCell = ({
 
     return (
         <Cell style={style}>
-            <Typography numberOfLines={1} variant="h5" bold style={styles.title}>
+            <Typography
+                numberOfLines={1}
+                variant="h5"
+                bold
+                style={!showSymbols && { paddingBottom: token.spacing.container.paddingVertical }}
+            >
                 {title}
             </Typography>
-            {showSymbols && (
-                <View style={[styles.iconListContainer, { flexDirection: symbolDirection }]}>
+            {showSymbols && iconsAndLabels.length > 0 && (
+                <View style={styles.iconListContainer}>
                     {iconsAndLabels.map((item, index) => (
                         <StatusIcon key={index} {...item} />
                     ))}
                 </View>
             )}
-            {maintenanceType && <Label label={maintenanceType} style={styles.label} />}
-            <PropertyList
-                workOrder={rest}
-                overwriteLabel={overwriteLabel}
-                valueColor={valueColor}
-                currentDate={currentDate.toDate()}
-            />
+            <View style={styles.dataContainer}>
+                <Typography group="paragraph" variant="body_short" color="textTertiary">
+                    {maintenanceType}
+                </Typography>
+                <PropertyRow label={workOrderType} value={workOrderId} />
+                <PropertyList workOrder={rest} valueColor={valueColor} currentDate={currentDate} />
```

Comment:
the `workOrder` prop on this component seems to a little bit misleading now that we are no longer passing the whole object in. Maybe this prop should be renamed to something more generic to help communicate that it's job is to take in any object that will be rendered in the list (if the key exists in the label mapping)